airflow/gcp/operators/video_intelligence.py

```diff
@@ -68,17 +71,17 @@ class CloudVideoIntelligenceDetectVideoLabelsOperator(BaseOperator):
 
     def __init__(
         self,
-        input_uri,
-        input_content=None,
-        output_uri=None,
-        video_context=None,
-        location=None,
-        retry=None,
-        timeout=None,
-        gcp_conn_id="google_cloud_default",
+        input_uri: str,
+        input_content: Optional[bytes] = None,
+        output_uri: Optional[str] = None,
+        video_context: Union[Dict, VideoContext] = None,
```

Comment:
```suggestion
        video_context: Optional[Union[Dict, VideoContext]] = None,
```
Also in other operators 🙊

Reply:
Fixed.